Rfam/Scripts/svn/rfci.pl

```diff
@@ -16,7 +16,8 @@
 #-------------------------------------------------------------------------------
 # Deal with all of the options
 
-my ( $message, @ignore, $preseed, $onlydesc, $help, $addToClan, $removeFromClan );
+my ( $message, @ignore, $preseed, $onlydesc, $help, $addToClan, $removeFromClan, $force_se_tag );
+$force_se_tag = undef;
```

Comment:
Explicitly setting `$force_se_tag = undef;` is redundant since it's already undef by default in the `my()` declaration?